.github/workflows/ci.yml

```
@@ -190,7 +190,7 @@ jobs:
 
   integration-tests:
     name: Cypress
-    if: ${{ !startsWith(github.head_ref, 'cf-preview/pr-') }}
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)